### PR TITLE
Changes admin permissions, Re-enables ghost ship story mode

### PR DESCRIPTION
**Pull request by improvedname**
Changes some of the admin perms, to more appropiate permission area, like mass edit being +debug(lets be honest don't trust a normal admin with mass edit 👅
Gives anything sound related its own tab so you can quick to it and stop sounds without finding a silly button 😠 

And also re-enables the "ghost ship" storyteller which is zero events, also require zero crew to start up.

### CODE REVIEW

**Comment by DogZeroX** [comment]
The sleeper isn't the ghost ship storyteller?

**Comment by improvedname** [comment]
The sleeper is calm storyteller low evens ghost ship is none at all

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -445,7 +445,7 @@ ADMIN_VERB_ADD(/client/proc/cmd_admin_add_freeform_ai_law, R_FUN, FALSE)
 		command_announcement.Announce("Ion storm detected near the ship. Please check all AI-controlled equipment for errors.", "Anomaly Alert", new_sound = 'sound/AI/ionstorm.ogg')
 
 
-ADMIN_VERB_ADD(/client/proc/cmd_admin_rejuvenate, R_ADMIN, FALSE)
+ADMIN_VERB_ADD(/client/proc/cmd_admin_rejuvenate, R_ADMIN|R_MOD, FALSE)
```
Comment (king5327) [inline code/modules/admin/verbs/randomverbs.dm]: Should Rejuvenate be a moderator button?

File: config/example/admin_ranks.txt [hunk 2]
```diff
@@ -27,7 +28,7 @@ Mentor          +MENTOR
 Admin Candidate	+ADMIN
 Trial Admin		+@ +BAN
 Badmin			+@ +SERVER +FUN
-Game Admin		+@ +DEBUG +PERMISSIONS
+Game Admin		+@ +DEBUG +PERMISSIONS + SOUND
```
Comment (king5327) [inline config/example/admin_ranks.txt]: Should be +SOUND, not + SOUND with the space between.
Also, move it up to badmin. This is *so* a badmin feature.

**Pull request merged**